sample-notificator-app/src/main/resources/notification-template/email-update-template.html

```diff
@@ -26,10 +26,17 @@
         </div>
       </li>
     </ul>
-    <p id="subscription-information">
-      You can also check the <a href="https://portal.qbic.uni-tuebingen.de/portal/" target="_blank">QBiC portal</a>
-      to learn more or <a id="unsubscription-link" href="" target="_blank">unsubscribe</a> from further notifications.
-    </p>
+    <div id="subscription-information">
+      <p> For more details on your samples visit the <a
+          href="https://portal.qbic.uni-tuebingen.de/portal/web/qbic/sample-tracking"
+          target="_blank">QBiC
+        portal.</a><br/>
```

Comment:
Just out of curiosity, why is here a line break?

Reply:
The `<br/>` is inserted as we are in the same paragraph. Paragraphs however have space between them which is avoided using this approach.